lib/takeaway.rb

```diff
@@ -0,0 +1,58 @@
+require_relative 'dish.rb'
+require_relative 'text.rb'
+require 'twilio-ruby'
+
+class Takeaway 
+  attr_reader :menu, :final_order, :texter
+  def initialize
+    @texter = Text.new
+    @menu = {
+      "dish 1" => 5,
+      "dish 2" => 7,
+      "dish 3" => 9}
+    @final_order = []
+  end
+
+  def menu
+    @menu.each do |dish, price|
+      puts "#{dish} -- £#{price}"
+    end
+  end
+
+  def add_dish(name, price)
+    dish = Dish.new(name, price)
+    @menu[dish.name] = dish.price
+  end
+
+  def selected_dishes(*order)
```

Comment:
I like the use of '*args' with "splat" operator, keeps the method flexible and open for expansion!